### ISSUE TITLE
Exceeding time out of the tracked BT beacon doesn't trigger away mode

### ISSUE THREAD

**Issue by keesdekraker** [heggink/domo-espresense#2]
First of all many thanks for building this great plugin!!

The plugin does not seem to trigger away mode when the tracked BT beacon is out of range.

When initializing the plugin while the beacon is not in range triggers away mode correctly.
But when the plugin is running and the beacon transitions from in range to out of range, the plugin does not trigger away mode. 

I'm running Domoticz build 2024.1 and Python 3.9.2.
Base stations (ESP32) run on Espresence v3.3.1

While I am no expert in Python, I fixed the issue by calling the 'prune_table' subroutine also within onHeartbeat routine (on top of the onPublish routine).  It then triggers correctly when the beacon is timed out after more then 10 seconds.

**Comment by heggink** [comment]
Hey @keesdekraker ,

Many thanks for you message. I stopped maintaining/using the plugin because I could not find an easy way to have different phones generate consistent beacons. I reached out to the domoticz app dev for android/ios but, despite incorporating it into the roadmap, never materialised whilst he switched to an annual fee model. The net is that everyone in my household would have to install bespoke apps to have a persistent beacon where I wold only want that within a geofence (as otherwise, everyone could track that beacon as well). So I stopped.

Happy to transfer onwership to you if you want.

Herman
